src/tools/fuzzing.h

```diff
@@ -1676,6 +1685,10 @@ class TranslateToFuzzReader {
   }
 
   Expression* makeConst(Type type) {
+    // There's no except_ref.const. Returns something else instead.
+    if (type == except_ref) {
+      return makeGetLocal(type);
```

Comment:
I think this can infinitely recurse :( if we have no local of that type, it will call this `makeConst` here...

Instead, one option is to add a new local of that type in makeGetLocal. Another is to have a singleton global which we use for this purpose.

Reply:
In the [reference type proposal](https://github.com/WebAssembly/reference-types/blob/master/proposals/reference-types/Overview.md), there's an instruction named `ref.null` which returns a nullref value. We can use that here after we implement those instructions, but can we leave it as TODO for now?